cmd/zqd/listen/command.go

```diff
@@ -79,6 +81,7 @@ func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
 	f.BoolVar(&c.pprof, "pprof", false, "add pprof routes to api")
 	f.BoolVar(&c.prom, "prometheus", false, "add prometheus metrics routes to api")
 	f.StringVar(&c.configfile, "config", "", "path to a zqd config file")
+	f.Var(&c.logLevel, "loglevel", "level for log output (defaults to info)")
```

Comment:
```suggestion
	f.Var(&c.logLevel, "loglevel", "level for log output")
```
because
```
$ go run ./cmd/zqd help listen |& grep loglevel
    -loglevel level for log output (defaults to info) (default "info")
```

Reply:
cool!